Formula/ehco.rb

```diff
@@ -0,0 +1,42 @@
+class Ehco < Formula
+  desc "Network relay tool and a typo :)"
+  homepage "https://github.com/Ehco1996/ehco"
+  url "https://github.com/Ehco1996/ehco/archive/refs/tags/v1.0.6.tar.gz"
+  sha256 "326c468c3790ad01031e52ccb4efcfa5e331d2198fdb13137749c67e8eaacf38"
+  license "GPL-3.0-only"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args, "cmd/ehco/main.go"
+  end
+
+  test do
+    version_info = shell_output("#{bin}/ehco -v")
+    assert_match(/ehco version */, version_info)
```

Comment:
```suggestion
    assert_match "ehco version", shell_output("#{bin}/ehco -v")
```
What format does this print version information? Depending on that, we can do something else, like:
```suggestion
    assert_match "ehco version v#{version}", shell_output("#{bin}/ehco -v")
```

Reply:
it's like `1.0.6`, can i use `assert_match(/ehco version *.*,*/, version_info)` to match?